tasties_app/tests/test_comment.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from tasties_app.models import Comment, Recipe
+from django.contrib.auth.models import User
+from django.utils import timezone
+from datetime import date
+
+
+@pytest.fixture
+def generate_comments():
+    """
+    in this generate function we create User, Category and Recipe
+    objects to fill Comment class and used to test
+     """
+    user = User.objects.create_user(username='test_user', password='password')
+    recipe = Recipe(title='Test Recipe1', author_id=user, description='Test Description1',
+                    directions='Test Directions1', publication_date=timezone.now(), minutes_to_make=1,
+                    recipe_picture='recipe.jpeg')
+    recipe.save()
+    the_date = date.today().strftime("%Y-%m-%d")
```

Comment:
There is no need for this variable. The Comment constructor will give the Comment object the current date if you do not specify any value for a date in the constructor arguments.